.github/workflows/simulation.yml

```diff
@@ -1,17 +1,14 @@
 name: simulate quicksilver
 
 on:
-  pull_request:
-    branches:
-      - main
+  schedule:
+    - cron: '0 0 * * 0' # run once a week
```

Comment:
can we do it based on some trigger? if we want to force run on some pe?

Reply:
So I moved the `run-sim` label trigger to `buildtest.yaml`.

This new scheduled job is an additional run of a really long simulation (more blocks and larger blocksize).

Reply:
Trigger still exists, just got moved to another file